zetaclient/tss/config.go

```diff
@@ -0,0 +1,116 @@
+package tss
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bnb-chain/tss-lib/ecdsa/keygen"
+	"github.com/multiformats/go-multiaddr"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	tsscommon "gitlab.com/thorchain/tss/go-tss/common"
+)
+
+const (
+	Port    = 6668
+	Version = "0.14.0"
+	Algo    = tsscommon.ECDSA
+)
+
+// MultiAddressFromString parses a string into a slice of addresses (for convenience).
```

Comment:
Seems the code coverage can be improved a bit in this file

Reply:
Agree. But this particular function is tested within libp2p package